Header header
geometry_msgs/Pose goal
int16 leader_id
int16 follower_num
int16[] follower_id
float32[] distance
float32[] angle
